Frontend/src/app/errorpages/error404/error404.component.html

```diff
@@ -0,0 +1,29 @@
+<html lang="en">
```

Comment:
you should write only div, not html, body and etc. Check how I did my components